packages/trader/src/Modules/Contract/Components/AccumulatorsStats/ticks-history-counter.tsx

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const TicksHistoryCounter = ({ has_progress_dots, value }) => (
+type TTicksHistoryCounter = {
+    dt_progress_dots?: string;
+    dt_ticks_history_counter?: string;
+    has_progress_dots: boolean;
+    value: number;
+};
+
+const TicksHistoryCounter = ({
+    dt_progress_dots,
+    dt_ticks_history_counter,
+    has_progress_dots,
+    value,
+}: TTicksHistoryCounter) => (
     <div
-        data-testid='dt_accu_stats_history_counter'
+        data-testid={dt_ticks_history_counter}
         className={classNames('accumulators-stats__history-counter', {
             'accumulators-stats__history-counter--emphasized': has_progress_dots && value === 0,
         })}
     >
         {value}
         {has_progress_dots && (
-            <div className='accumulators-stats__progress-dots'>
+            <div className='accumulators-stats__progress-dots' data-testid={dt_progress_dots}>
```

Comment:
oh, by naming convention I meant passing them like this:
`ticks_history_counter_testid='dt_ticks_history_counter'` and `progress_dots_testid='dt_progress_dots'`
and in this file:
`data-testid='dt_ticks_history_counter'`
`data-testid='dt_progress_dots'`